pkg/codegen/testing/test/testdata/output-funcs/nodejs/funcWithEmptyOutputs.ts

```diff
@@ -21,19 +21,3 @@ export interface FuncWithEmptyOutputsArgs {
      */
     name: string;
 }
-
-export interface FuncWithEmptyOutputsResult {
-}
-/**
- * n/a
- */
-export function funcWithEmptyOutputsOutput(args: FuncWithEmptyOutputsOutputArgs, opts?: pulumi.InvokeOptions): pulumi.Output<FuncWithEmptyOutputsResult> {
-    return pulumi.output(args).apply((a: any) => funcWithEmptyOutputs(a, opts))
-}
-
-export interface FuncWithEmptyOutputsOutputArgs {
-    /**
-     * The Name of the FeatureGroup.
-     */
-    name: pulumi.Input<string>;
-}
```

Comment:
Why did we remove the `*Output` function type for this? (Same with other languages).

Reply:
`*Output` functions are not generated when `Outputs == nil` (this is current behaviour) I have extended the behaviour such that empty outputs is the same thing as no outputs which is why these functions are removed